zerver/openapi/zulip.yaml

```diff
@@ -6793,6 +6793,19 @@ paths:
                 items:
                   type: integer
               example: [1, 2, 3]
+        - name: allow_empty_topic_name
+          in: query
+          description: |
+            Whether the topic names (`subject` field, and `prev_topic`, `topic` in `edit_history` objects)
+            in the messages returned can be empty string.
+
+            If `false`, `"general chat"` is returned replacing `""` as the topic name.
```

Comment:
> Whether the client supports processing the empty string as a topic in the topic name fields in the returned data, including in returned `edit_history` data. 

> If `false`, the server will use `"general chat"` instead of `""` to represent the empty string topic in its response.

And add to the changes entry:

> Previously, the empty string was not a valid topic.

Reply:
Done